package.json

```diff
@@ -34,44 +34,46 @@
     "prettier": "prettier --write .",
     "build": "rollup -c",
     "release": "npm run build && np",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6006 -h localhost",
```

Comment:
Fair warning that this might need a `cross-env` prefix for windows (so it's a warning for me 😀)